.github/workflows/build.yml

```diff
@@ -192,11 +180,32 @@ jobs:
         Add-Content -Path "$Env:GITHUB_ENV" -Value "WINDOWS_CODESIGN_CERTIFICATE_PASSWORD=$Env:WINDOWS_CODESIGN_CERTIFICATE_PASSWORD"
         Add-Content -Path "$Env:GITHUB_ENV" -Value "CMAKE_ARGS_EXTRA=-DWINDOWS_CODESIGN=ON"
 
-    - name: "Set up build environment"
-      id: buildenv
-      run: ${{ matrix.buildenv_script }} setup
-      env:
-        BUILDENV_BASEPATH: ${{ matrix.buildenv_basepath }}
+    - name: Install Nuget
+      uses: nuget/setup-nuget@v1
+      if: runner.os != 'Linux'
+      with:
+        nuget-version: 5.10.0
+
+    - name: "Authenticate to GitHub Packages"
+      if: runner.os != 'Linux' && github.repository_owner == 'mixxxdj'
+      shell: bash
+      run: |
+        nuget sources add -Name mixxx -Source https://nuget.pkg.github.com/mixxxdj/index.json -Username mixxxdj -Password ${{ secrets.GITHUB_TOKEN }} -StorePasswordInClearText -ForceEnglishOutput -NonInteractive
+        nuget setapikey ${{ secrets.GITHUB_TOKEN }} -Source mixxx -ForceEnglishOutput -NonInteractive
+        echo "VCPKG_BINARY_SOURCES=clear;nuget,mixxx,readwrite;" >> "${GITHUB_ENV}"
+
+    - name: Get Git commit of vcpkg submodule
+      if: runner.os != 'Linux'
+      shell: bash
+      run: echo VCPKG_COMMIT=$(git ls-tree HEAD vcpkg | awk '{print $3}') >> "${GITHUB_ENV}"
```

Comment:
We should not reuse the VCPKG prefix.

Reply:
Why not?

Reply:
Please suggest another name then.

Reply:
> Why not?

I have started to craw the VCPKG manuals to find out what this is for. We should release others from this work. 

Let's drop the idea of using submodules ..

Reply:
How about MIXXX_VCPKG_SUBMODULE_COMMIT?

Reply:
Please read the commit form a file per target, and then check out the package with a normal git command. 
This maintains all flexibility and should fullfill your needs as well without introducing the issues known form git submodules.

Reply:
this would be MIXXX_VCPKG_COMMIT

Reply:
That would be reimplementing Git submodule but hackier.

Reply:
How do target the issue if we need different hashes for different targets? 

Using a repository with submodules is not trivial, I really like to avoid the related headaches for all of us. Git submodules is oversized to just download a folder.

Reply:
> How do target the issue if we need different hashes for different targets?

We don't do that. I consider that a problematic aspect of our current setup. If one OS needs something different from another, then that needs to be specified either in vcpkg.json or using an overlay. I do not think maintaining separate branches of the vcpkg repo for different OSes is a good idea.